alacritty/src/event.rs

```diff
@@ -413,7 +417,9 @@ impl<N: Notify> Processor<N> {
 
                     let size = self.display.window.inner_size().to_physical(dpr);
 
-                    display_update_pending.font = Some(self.font.clone());
+                    if display_update_pending.font.is_none() {
+                        display_update_pending.font = Some(self.config.font.clone());
+                    }
```

Comment:
Why do we need to check for `None` here?

Reply:
I already removed it by the time you commented; my thinking was that `display_update_pending.font` might already be `Some` from  earlier in the event handling loop but then I had a closer look at the logic for this block and realized that won't ever happen.